server/repository/src/migrations/v2_02_01/mod.rs

```diff
@@ -0,0 +1,135 @@
+use super::{version::Version, Migration, MigrationFragment};
+
+use crate::StorageConnection;
+
+mod add_store_ids_to_existing_rnr_form_changelogs;
+
+pub(crate) struct V2_02_01;
+
+impl Migration for V2_02_01 {
+    fn version(&self) -> Version {
+        Version::from_str("2.2.1")
+    }
+
+    fn migrate(&self, _connection: &StorageConnection) -> anyhow::Result<()> {
+        Ok(())
+    }
+
+    fn migrate_fragments(&self) -> Vec<Box<dyn MigrationFragment>> {
+        vec![Box::new(
+            add_store_ids_to_existing_rnr_form_changelogs::Migrate,
+        )]
+    }
+}
+
+#[cfg(test)]
+#[actix_rt::test]
+async fn migration_2_02_01() {
+    use v2_02_00::V2_02_00;
+
+    use crate::migrations::*;
+    use crate::test_db::*;
+
+    let previous_version = V2_02_00.version();
+    let version = V2_02_01.version();
+
+    let SetupResult { connection, .. } = setup_test(SetupOption {
+        db_name: &format!("migration_{version}"),
+        version: Some(previous_version.clone()),
+        ..Default::default()
+    })
+    .await;
+
+    add_rnr_form_changelogs_no_store_id(&connection).unwrap();
+
+    assert_eq!(check_changelogs_have_store_id(&connection), false);
+
+    // Run this migration
+    migrate(&connection, Some(version.clone())).unwrap();
+    assert_eq!(get_database_version(&connection), version);
+
+    assert_eq!(check_changelogs_have_store_id(&connection), true);
+}
+
+#[cfg(test)]
+fn check_changelogs_have_store_id(connection: &StorageConnection) -> bool {
+    use crate::{ChangelogFilter, ChangelogRepository, ChangelogTableName};
+
+    let mut changelogs = ChangelogRepository::new(connection)
```

Comment:
If changelog repo every changes (new field), this test would fail. Probably the faster is to create a temp table to this:

https://github.com/msupply-foundation/open-msupply/blob/8b5fbfde1ef930ecf8ea20a218c90ba2488ca97a/server/repository/src/migrations/templates/data_migration/mod.rs#L152-L156

https://github.com/msupply-foundation/open-msupply/blob/8b5fbfde1ef930ecf8ea20a218c90ba2488ca97a/server/repository/src/migrations/templates/data_migration/mod.rs#L18-L23